openslides/core/static/js/core/site.js

```diff
@@ -107,6 +107,38 @@ angular.module('OpenSlidesApp.core.site', [
     }
 ])
 
+// Set up the activeApp for the title from the webpage
+.run([
+    '$rootScope',
+    'gettextCatalog',
+    function ($rootScope, gettextCatalog) {
+        $rootScope.activeApp = '';
+        $rootScope.$on('$stateChangeSuccess', function(evt, toState) {
+            console.log(toState);
+            if (toState.name.startsWith('core.tag')) {
+                $rootScope.activeApp = gettextCatalog.getString('Tags');
+            } else {
+                var state = toState.name.split('.')[0];
+                switch (state) {
```

Comment:
@emanuelschuetze Do we have existing code for getting the verbose app name from state to replace this?